server/models/Diary.js

```diff
@@ -1,4 +1,35 @@
+import { required, minLength, dataType } from '../helpers/utils';
+
 const diaries = [];
+
+const postRules = {
+  title: [
+    [required],
+    [minLength, 5],
```

Comment:
These could have been objects
```
{minLength: 5}
```
But if this works fine for you then great

Reply:
I was going to explain why I used the [function: arg] syntax.
The {function: arg} syntax might be more difficult and might take me longer to implement (and I have a lot left to do).
Yes, it works fine for now.
I'll probably revisit it later, when I have more time.
Thank you for the review.